### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by adrianvintu** [xqwzts/flutter_circular_chart#27]
I see the last version was uploaded on Aug 30, 2018

https://pub.dev/packages/flutter_circular_chart/versions

**Comment by LucasBSousa** [comment]
Yeah, really wish that stroke width PR would be merged. It's been sitting still for almost a whole year already.

**Comment by adrianvintu** [comment]
Looks like it was forked here https://pub.dev/packages/flutter_circular_chart_two

I did not test the fork.
